integration_tests/dbt_project.yml

```diff
@@ -24,7 +24,7 @@ vars:
 
 models:
   fivetran_log:
-    +schema: fivetran_platform
+    +schema: "{{ 'sqlw_tests' if target.name == 'databricks-sql' else 'fivetran_platform' }}"
```

Comment:
This is an artifact of needing to use two different schemas for the Databricks jobs